public/app/plugins/datasource/cloudwatch/components/LogGroups/LogGroupsSelector.tsx

```diff
@@ -178,8 +193,25 @@ export const LogGroupsSelector = ({
           </div>
         </div>
         <Space layout="block" v={2} />
+        <EditorField label="Template variable" width={26}>
+          <Select
+            isClearable
+            aria-label="Template variable"
+            value={currentVariableOption}
+            allowCustomValue
+            options={variableOptions}
+            onChange={(option) => {
+              const newValues = selectedLogGroups.filter((lg) => !lg.name?.startsWith('$'));
+              if (option?.label) {
+                newValues.push({ name: option.label, arn: option.label });
+              }
+              setSelectedLogGroups(newValues);
+            }}
+          />
+        </EditorField>
+        <Space layout="block" v={2} />
         <Label className={styles.logGroupCountLabel}>
-          {selectedLogGroups.length} log group{selectedLogGroups.length !== 1 && 's'} selected
+          {selectedLogGroupsCounter} log group{selectedLogGroupsCounter !== 1 && 's'} selected
```

Comment:
I do sort of understand why the variables don't count toward this, but also I feel like it may be confusing from a user perspective if their variable maps to a log group and isn't counted?

Reply:
Catherine wanted variables to be excluded from this counter. But I remember now that she also me to move this counter so that it's placed above the template variable field. That way it's a bit more obvious that the counter is only for non-variable selections.